src/shared/analytics/shared/helpers.ts

```diff
@@ -0,0 +1,29 @@
+import type { Asset } from 'defi-sdk';
+import { isTruthy } from 'is-truthy-ts';
+import { getDecimals } from 'src/modules/networks/asset';
+import type { Chain } from 'src/modules/networks/Chain';
+import { baseToCommon } from 'src/shared/units/convert';
+
+export function toMaybeArr<T>(
+  arr: (T | null | undefined)[] | null | undefined
+): T[] | undefined {
+  return arr?.length ? arr.filter(isTruthy) : undefined;
```

Comment:
```suggestion
  return arr?.filter(isTruthy) ?? undefined;
```

Reply:
Thank you! I copied the previous implementation and didn't change it. However, I can try to use this version

Reply:
ah, didn't notice it was like this before :)